rq/worker.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+#wait_time_after_connection_failure -*- coding: utf-8 -*-
```

Comment:
Why does this line need to be changed?

Reply:
I visited this thread from the issue #998 
may I know what is the reason, this PR is not merged yet into master ? I can make required changes & provide a fix.

rq workers are **not automatically retrying** to connect when connect is lost/redis-server is restarted for maintenance. So until rq worker is restarted, it keeps raising ConnectionError resulting in failure. Lately this is becoming serious issue in production. Hoping this PR can fix it.

Reply:
My mistake!